v21.2/create-changefeed.md

```diff
@@ -202,9 +202,42 @@ Kafka has the following topic limitations:
 - Topics with a period (`.`) and underscore (`_`) can collide on internal Kafka data structures, so you should use either but not both.
 - Characters not accepted by Kafka will be automatically encoded as unicode characters by CockroachDB.
 
-## Advanced Configuration
+## Advanced changefeed configuration
 
-### Kafka Sink Configuration
+{{site.data.alerts.callout_danger}}
+The configurations and settings explained in these sections will have significant impact on a changefeed's behavior.
+{{site.data.alerts.end}}
+
+The following sections describe settings, configurations, and details to tune changefeeds for particular use cases:
+
+* [High durability delivery](#tuning-for-high-durability-delivery)
+* [High throughput](#tuning-for-high-throughput)
+* [Kafka sinks](#kafka-sink-configuration)
+* [Webhook sinks](#webhook-sink-configuration)
+
+### Tuning for high durability delivery
+
+When designing a system that relies on high durability of message delivery — that is, not missing any message acknowledgement at the downstream sink — consider the following settings and configuration. Before tuning these settings we recommend reading details on our [changefeed at-least-once-delivery guarantee](stream-data-out-of-cockroachdb-using-changefeeds.html#ordering-guarantees).
+
+* Increase the number of seconds before [garbage collection](architecture/storage-layer.html#garbage-collection) with the [`gc.ttlseconds`](configure-replication-zones.html#gc-ttlseconds) setting to provide a higher recoverability window for data if a changefeed fails. For example, if a sink is unavailable, changes are queued until the sink is available again. While the sink is unavailable, changes will be retried until the garbage collection window is reached and then the data is removed.
+  * You can also use the [`protect_data_from_gc_on_pause`](#protect-pause) option in combination with [`on_error=pause`](#on-error) to explicitly pause a changefeed on error and to then protect the changes from garbage collection.
```

Comment:
I would say instead of going into a failure state, because if we have a retryable error we will still retry and not pause